spec/metrics/relation_spec.rb

```diff
@@ -240,30 +234,25 @@
 
   describe "#delete_all" do
     it do
-      Timecop.freeze(Time.now) do
-        expect(rel.where(user_id: 1, dummy: 'q', timer: Time.now).delete_all)
-          .to eq "delete from \"dummy\" where (user_id=1) and (dummy='q') and (timer=#{Time.now.to_i}s)"
-      end
+      Timecop.freeze(Time.now)
+      expect(rel.where(user_id: 1, dummy: 'q', timer: Time.now).delete_all)
+        .to eq "delete from \"dummy\" where (user_id=1) and (dummy='q') and (timer=#{Time.now.to_i}s)"
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.